scripts/travisci/check_precommit.sh

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env bash
 status=0
 
-echo "Checking commit range ${TRAVIS_COMMIT_RANGE}"
-SOURCE="${TRAVIS_COMMIT_RANGE%...*}"
-ORIGIN="${TRAVIS_COMMIT_RANGE#*...}"
+echo "Checking commit range ${COMMIT_RANGE}"
+SOURCE="${COMMIT_RANGE%...*}"
```

Comment:
doesn't this break travis?

Reply:
oh wait, I forgot we still had it. Would it be better if we changed the environment variable in github actions for the time being?